Android/src/org/droidplanner/android/proxy/mission/item/fragments/MissionDetailFragment.java

```diff
@@ -206,11 +208,69 @@ public void onSpinnerItemSelected(Spinner spinner, int position) {
         }
     };
 
+    private final Spinner.OnItemSelectedListener missionFrameListener = new Spinner.OnItemSelectedListener() {
+
+        @Override
+        public void onItemSelected(AdapterView<?> parent, View view, int position, long id) {
+            frameSpinner = (Spinner) view.findViewById(R.id.frameSpinner);
```

Comment:
@billbonney You should be able to remove this assignment since `frameSpinner` is already set in `setupFrameSpinner()`.